tests/properties/pflua_pipelines_match.lua

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env luajit
+-- -*- lua -*-
+module(..., package.seeall)
+package.path = package.path .. ";../?.lua;../../src/?.lua"
+-- Compare the results of the libpcap/bpf and pure-lua pflua pipelines.
+
+local pf = require("pf")
+local savefile = require("pf.savefile")
+local utils = require('pf.utils')
+
+local pflang = require('pfquickcheck.pflang')
+
+function property(packets)
+   --nil pkt_idx, pflang_expr, bpf_result, pflua_result to avoid
+   -- confusing debug information
+   pkt_idx, pflang_expr, bpf_result, pflua_result = nil
+   local pkt, P, pkt_len, libpcap_pred, pflua_pred
+   a = pflang.Pflang()
+   pflang_expr = table.concat(a, ' ')
```

Comment:
i think probably we need to generate the string in a more clever way :)  that way we respect precedence rules.  a useful early step would be to just slather the thing in extra parentheses, parentheses everywhere, and then later do something more minimal.